### PR TITLE
chore(release): version bump

### PR DESCRIPTION
**Pull request by changxu0306**
Bump version from 2.22.2 to 2.22.3 for release.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -25,8 +25,8 @@ android.useAndroidX=true
 android.enableJetifier=true
 
 GROUP=com.amazonaws
-VERSION_NAME=2.22.2
-VERSION_CODE=022202
+VERSION_NAME=2.22.3
+VERSION_CODE=022203
```
Comment (richardmcclellan) [inline gradle.properties:29]:
```suggestion
```

Comment (raphkim) [reply gradle.properties:29]: @richardmcclellan, do we not use the version code anymore?

Comment (richardmcclellan) [reply gradle.properties:29]: @raphkim we [removed it from amplify-android](https://github.com/aws-amplify/amplify-android/pull/1223), and the change was released yesterday.    My understanding is that it is not needed at all for an Android library - it's only needed for an Android app.

Comment (changxu0306) [reply gradle.properties:29]: If we want to remove `VERSION_CODE`, I think it's better to put another PR.

**Review by richardmcclellan — COMMENTED** [review]
Overall this looks :+1:, but unless these are urgent changes, could we wait until Monday morning to release?  Just to avoid any issues that may require attention over the weekend!

**Comment by changxu0306** [comment]
> Overall this looks 👍, but unless these are urgent changes, could we wait until Monday morning to release? Just to avoid any issues that may require attention over the weekend!

Sure thing! Before that, I can also put a PR for removing `VERSION_CODE`.

File: gradle.properties (hunk 1 above)
Comment (richardmcclellan) [reply gradle.properties:29]: @changxu0306 okay! https://github.com/aws-amplify/aws-sdk-android/pull/2388

Comment (richardmcclellan) [reply gradle.properties:29]: I just checked the [AppSync repo's `gradle.properties`](https://github.com/awslabs/aws-mobile-appsync-sdk-android/blob/main/gradle.properties), and it doesn't even have `VERSION_CODE`!    So, that further confirms removing it is fine.

**Review by raphkim — APPROVED** [review]

**Review by richardmcclellan — APPROVED** [review]

**Pull request merged**